### ISSUE TITLE
get `idx_to_labels` values in Image Classification datamodule

### ISSUE THREAD

**Issue by aniketmaurya** [PyTorchLightning/lightning-flash#621]
## 🚀 Feature
it would be helpful if we can have either of these two methods in Flash Dataloader:
- provide label encoding function/mapping while creating the Data module
- or get the mapping

### for example

```python
data_module = DataModule.from_csv(
    "input",
    "target",
    train_file="train_data.csv",
    train_transform={
        "to_tensor_transform": torch.as_tensor,
    },
)
```
`print(data_module.idx_to_labels)` should give me the mapping

**Comment by aniketmaurya** [comment]
hi @ethanwharris, it would be very helpful to have this feature. Any suggestion so that maybe I can start on this one?
